internal/http/certificate.go

```diff
@@ -0,0 +1,22 @@
+package http
+
+import (
+	"crypto/x509"
+	"errors"
+	"os"
+)
+
+func LoadRootCAs(path string) (*x509.CertPool, error) {
```

Comment:
From functionality wise, it is better to be named as `LoadCertPool()` as it can be used for intermediate certs.